### PR TITLE
Improve deletion flow consistency

### PR DESCRIPTION
**Pull request by callumacrae**
### Context

There were a couple different designs for the deletion pages before, this makes them all looks the same and (hopefully) makes them look a bit nicer

### Changes proposed in this pull request

Button groups

Also, there was no flash message after deleting a delivery provider - I fixed that

### Guidance to review

Percy would help here :D

### Testing

yes

### CODE REVIEW

File: app/views/admin/administrators/administrators/delete.html.erb
Comment (callumacrae) [inline app/views/admin/administrators/administrators/delete.html.erb:2]: I didn't really know what else to write here, but `Admin user: [user]` felt clunky to me

Comment (kicferk1) [reply app/views/admin/administrators/administrators/delete.html.erb:2]: `Once deleted, <%= @administrator.full_name %>'s account won't be recoverable`?

Comment (callumacrae) [reply app/views/admin/administrators/administrators/delete.html.erb:2]: Hah, it's the `won't be recoverable` I thought was a bit weird (isn't it technically not even true?), but yeah, that's a good suggestion

**Review by kicferk1 — APPROVED** [review]

File: app/views/admin/administrators/administrators/delete.html.erb
Comment (kicferk1) [reply app/views/admin/administrators/administrators/delete.html.erb:2]: Well, there is no way for the user to recover it yet, soooo, it's not technically lying :D

**Review by ethanmills — CHANGES_REQUESTED** [review]
This was specified by Shani, please check this thread before changing anything: https://ukgovernmentdfe.slack.com/archives/C01RSEKR56C/p1615559271010100

**Comment by callumacrae** [comment]
Have changed them to match the copy from that slack thread - but the buttons should definitely be next to each other!

File: app/views/admin/administrators/administrators/delete.html.erb
Comment (seb-siem-softwire) [reply app/views/admin/administrators/administrators/delete.html.erb]: This was exact text that Shani had requested - double-check with her if the text is correct

File: app/views/admin/administrators/administrators/delete.html.erb
Comment (seb-siem-softwire) [inline app/views/admin/administrators/administrators/delete.html.erb:1]: Double check with Shani if the messaging is right. As long as she is happy it should be ok

File: app/views/admin/administrators/administrators/edit.html.erb
Comment (seb-siem-softwire) [inline app/views/admin/administrators/administrators/edit.html.erb:5]: Double check with Shani

File: spec/cypress/integration/admin/administrators/delete-administrator-user.js
Comment (kicferk1) [inline spec/cypress/integration/admin/administrators/delete-administrator-user.js]: Hehe, don't you wish we had visual tests that you would have to recreate instead of this? :D

Comment (callumacrae) [reply spec/cypress/integration/admin/administrators/delete-administrator-user.js]: I wouldn't have needed to change it three times in this PR so far if they were visual tests because visual tests only need approving once! (and they're approved by pressing a button, not by having to modify a test file)

**Review by ethanmills — APPROVED** [review]

File: spec/cypress/integration/admin/administrators/delete-administrator-user.js
Comment (kicferk1) [reply spec/cypress/integration/admin/administrators/delete-administrator-user.js]: Oh, I thought they work more like snapshot tests. Interesting, now I hope we get them even more.

**Comment by callumacrae** [comment]
I've reverted all the changes but the button groups and missing flash message, this PR wasn't worth the amount of time it was taking to get it through the review process

**Review by amazimbe — APPROVED** [review]

**Review by seb-siem-softwire — APPROVED** [review]

**Pull request merged**
